### ISSUE TITLE
Cannot build windows10 appx.

### ISSUE THREAD

**Issue by hv0905** [pwa-builder/PWABuilder-CLI#336]
CLI version: 2.0.3-rc.1
I have tried the following method to generate an windows 10 appx with pwabuilder,.
```
pwabuilder https://web.kahla.app/ -i path_to_icon.png -p windows10,web -l debug
```
the pwabuilder printed:
```
 ...
[debug] pwabuilder  : Loading platform module: pwabuilder-windows10
[debug] pwabuilder  : Loading platform module: pwabuilder-web
 ...
```
which shows that it should loaded the pwabuilder-windows10 correctly.
Then i execute:

```
cd Kahla
pwabuilder package -p windows10 -l debug
```
But the pwabuilder just do nothing and print a debug output: 
```
[debug] pwabuilder  : Available platforms within project: web
[debug] pwabuilder  : Packaging the following platforms: 
```
I don't know why the windows10 platform is unavaliable and I wonder if it is because of a bug of pwabuilder or my fault.

**Comment by JudahGabriel** [comment]
Thanks for the error report. We'll investigate and get back to you.

**Comment by JudahGabriel** [comment]
Are you guys able to try packaging with pwabuilder.com? Put in your URL -> Build My App -> Microsoft Store. Can you try?

**Comment by hv0905** [comment]
Yes, I have already packed a PWA appx via `pwabuilder.com` and uploaded it to MS Store successfully.

**Comment by fasouto** [comment]
Same here, I'm able to package via pwabuilder.com. Unfortunately I need the CLI since I need to change the `DisplayName` and I cannot do that on the website.

**Comment by JudahGabriel** [comment]
Can you confirm whether you have the required Win10 images: 44x44, 50x50, 150x150? I'm wondering if it's related to [this issue](https://github.com/pwa-builder/PWABuilder/issues/552).
